students/jeff_shabani/session03/list_lab.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+
+
+FRUIT_LIST = ['Apples', 'Pears', 'Oranges', 'Peaches']
+
+
+def series_one(iterable):
+    print(iterable)
```

Comment:
I know that your just passing in the fruit list, but `iterable` doesn't describe what the object is at all.  Just call it `fruit_list` here.